### PR TITLE
Added walls and floors made of meat

### PR DESCRIPTION
**Pull request by Useroth**
Also it's easily copypastable to make turf paths for any other material if needed

### CODE REVIEW

**Comment by tf-4** [comment]
what

you know this already exists, and we disabled it for it being too LRP, right

**Comment by Useroth** [comment]
> what
> 
> you know this already exists, and we disabled it for it being too LRP, right

They still cannot be obtained through normal means, this is for the purposes of being able to spawn them in buildmode or map them.

**Comment by Useroth** [comment]
_(Also probably DNM for now, since the integration tests fail. This PR can be testmerged for the purposes of an event.)_

File: modular_skyrat/master_files/code/game/turfs/materialfloors.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/turf/open/floor/material/meat
+	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT / 4)
```
Comment (tf-4) [inline modular_skyrat/master_files/code/game/turfs/materialfloors.dm]:
```suggestion
	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT * 0.25)
```

**Comment by Gandalf2k15** [comment]
ree this was disabled, just renable it

**Comment by Gandalf2k15** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/2540

**Comment by GoldenAlpharex** [comment]
> ree this was disabled, just renable it

That doesn't change anything, material walls can't be spawned in directly or used in mapping without a PR like this one, it's entirely separate trains of thought.

**Comment by Iamgoofball** [comment]
Just turn the normal meat and pizza walls back on. It's not like anyone will notice and it avoids this nonsense while keeping the mechanics of said food walls.

**Comment by GoldenAlpharex** [comment]
> Just turn the normal meat and pizza walls back on. It's not like anyone will notice and it avoids this nonsense while keeping the mechanics of said food walls.

Once again irrelevant, you literally can't map any of the material walls, this is for mapping for an event, specifically.

**Comment by gonenoculer5** [comment]
> > Just turn the normal meat and pizza walls back on. It's not like anyone will notice and it avoids this nonsense while keeping the mechanics of said food walls.
> 
> Once again irrelevant, you literally can't map any of the material walls, this is for mapping for an event, specifically.

It smells of necromorphs in this house. Damn.

**Comment by hugmeorelse** [comment]
Why do we even need this?

**Comment by tf-4** [comment]
> Why do we even need this?

As explained above, "this is for mapping for an event, specifically."

**Comment by tf-4** [comment]
Some of this can go upstream, rest is unneeded

**Pull request closed without merging**